src/cloud.ts

```diff
@@ -1,13 +1,15 @@
 import * as exec from '@actions/exec'
 import * as spin from './spin'
 
+export const DEFAULT_CLOUD_URL = "https://cloud.fermyon.com"
+
 export async function login(cloudBaseURL: string, token: string): Promise<void> {
     await exec.exec('spin', ['cloud', 'login', '--url', cloudBaseURL, '--token', token])
 }
 
-export async function deploy(appConfigFile: string): Promise<Metadata> {
-    const manifest = spin.getAppConfig(appConfigFile)
-    const result = await exec.getExecOutput("spin", ["deploy", "-f", appConfigFile])
+export async function deploy(manifestFile: string): Promise<Metadata> {
+    const manifest = spin.getAppManifest(manifestFile)
```

Comment:
Not sure where this is used?

Reply:
this is used to get name of the app, which is then used for parsing logs to fetch route info

Reply:
Doh, I forgot to expand the context; it is of course used ONE LINE below what GitHub showed in the preview.  Thanks for explaining!